apps/spruce/src/pages/task/taskTabs/TaskHistory/CommitDetailsCard/index.tsx

```diff
@@ -153,6 +159,8 @@ const CommitCard = styled.div<{ status: TaskStatus }>`
   border: 1px solid ${gray.light2};
 
   ${({ status }) => `border-left: ${size.xs} solid ${statusColorMap[status]};`}
+
+  ${({ isMatching }) => !isMatching && "opacity: 0.5;"}
```

Comment:
We have this style that can possibly be used?
https://github.com/evergreen-ci/ui/blob/3c5d36a8708b71653bfd48cd23ac6c987f7936c1/apps/spruce/src/components/styles/inactive.ts#L3